README.md

```diff
@@ -1,6 +1,6 @@
 ## yaxpeax-x86
 
-x86 decoders implemented as part of the yaxpeax project.
+x86 decoders implemented as part of the yaxpeax project it is a mazing projec.
```

Comment:
I regret the spam.

Reply:
i regret being alive

Reply:
you're alive forever in the commit history

Reply:
oh no